ci/wasm_build.py

```
@@ -457,36 +457,76 @@ def _escape_meson_string(value: str) -> str:
+
+        system = _platform_mod.system().lower()
+        platform_name = (
+            "win"
+            if system == "windows"
+            else ("darwin" if system == "darwin" else "linux")
+        )
+        machine = _platform_mod.machine().lower()
+        arch = "x86_64" if machine in ("x86_64", "amd64") else "arm64"
+        ensure_emscripten_available(platform_name, arch)
+        _emscripten_patch_applied = True
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)